src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -8,12 +8,36 @@
 from vivarium_census_prl_synth_pop.constants import paths
 
 
-def build_results(results_dir: str) -> None:
+def build_results(results_dir: str, mark_best: bool, test_run: bool) -> None:
+    if mark_best and test_run:
+        logger.error(
+            "A test run can't be marked best. "
+            "Please remove either the mark best or the test run flag."
+        )
+        return
+
     logger.info("Creating final results directory.")
-    final_output_dir = utilities.build_output_dir(
+    final_output_root_dir = utilities.build_output_dir(
         Path(results_dir), subdir=paths.FINAL_RESULTS_DIR_NAME
-    ) / datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
+    )
+    final_output_dir = final_output_root_dir / datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
 
     logger.info("Copying raw results to final location.")
     raw_output_dir = Path(results_dir) / paths.RAW_RESULTS_DIR_NAME
     shutil.copytree(raw_output_dir, final_output_dir)
+
+    if test_run:
+        logger.info("Test run - not marking results as latest.")
```

Comment:
Do we want test runs to default to the usual output dir? I wonder if we specifically DON'T want that.

Or rather...what are you envisioning a "test run" is used for?

Reply:
+ 1.  I usually save test runs to my own folder.  I do like the flexibility to do a run and save it to the team folder and not mark it as latest or best.

Reply:
Some clarifications
1. Latest happens automatically. You don't do anything for that.
2. I agree we need to be able to save to team folder and not mark best. That's exactly what `make_results` would do by default. But I'm wondering in what scenario do we want to `make_results -t` to the official folder?

Reply:
A test run is a run that outputs to the same location, but doesn't mark latest. That's all. Maybe we never use it.

Reply:
oh, i see now. I guess that's just a fundamentally different mental model than what I had in mind of what "latest" meant.